### ISSUE TITLE
Socketed Attack/Spells do <X> Parsed Incorrectly?

### ISSUE THREAD

**Issue by Lothrik** [PathOfBuildingCommunity/PathOfBuilding#1273]
Currently, modifiers like these:
* Socketed Spells deal 20% more Damage
* Socketed Spells have +3% to Critical Strike Chance

.. will increase the damage dealt by Zombies and Summon Raging Spirits in Path of Building.

But in-game if you want to increase the damage dealt by Zombies or Summon Raging Spirits, you (apparently) need modifiers like these:
* Socketed Attacks deal 20% more Damage
* Socketed Attacks have +3% to Critical Strike Chance

See: https://www.reddit.com/r/pathofexile/comments/9jla16/psa_socketed_attacksspells_deal_x_more_damage_is/

For obvious reasons this needs significantly more testing and verification, I just thought it should be brought up as the current way PoB handles things seems to contradict in-game testing.

**Comment by Lothrik** [comment]
I also suggest reading the comments in that thread, as there's a lot more useful testing information present:

"(also heard CA degen ground doesn't work with either? need to confirm but can't atm, which would mean things like toxic rain probably dont either) Correct, I found this out the hard way. At least I now have a pretty good ele hit bow."

"The subterranean mod that can roll on weapons states, Socketed Spells do 40% More Damage.
Heralds do not work with the mod. I suggest changing the wording to Socketed Spells do 40% more Spell Damage."

"I made a +2 vicious projectile bow with socketed attacks deal 20% more damage, wasn't very happy to find out it didnt even work (toxic rain)"

(.. and so on.)
